lib/smack/AnnotateLoopExits.cpp

```diff
@@ -0,0 +1,96 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+
+//
+// This pass normalizes loop structures to allow easier generation of Boogie
```

Comment:
This comment seems to be copy-pasted from a previous pass. Update it or remove it.